platform/iphone/gl_view.mm

```diff
@@ -198,62 +53,32 @@ CGFloat _points_to_pixels(CGFloat points) {
 	return (points / pointsPerInch * pixelPerInch);
 }
 
-@implementation GLView
-
-@synthesize animationInterval;
-
-static const int max_touches = 8;
-static UITouch *touches[max_touches];
-
-static void init_touches() {
-
-	for (int i = 0; i < max_touches; i++) {
-		touches[i] = NULL;
-	};
-};
-
-static int get_touch_id(UITouch *p_touch) {
-
-	int first = -1;
-	for (int i = 0; i < max_touches; i++) {
-		if (first == -1 && touches[i] == NULL) {
-			first = i;
-			continue;
-		};
-		if (touches[i] == p_touch)
-			return i;
-	};
-
-	if (first != -1) {
-		touches[first] = p_touch;
-		return first;
-	};
-
-	return -1;
-};
-
-static int remove_touch(UITouch *p_touch) {
+OS::VideoMode _get_video_mode() {
+	int backingWidth;
+	int backingHeight;
+	glGetRenderbufferParameterivOES(GL_RENDERBUFFER_OES,
+			GL_RENDERBUFFER_WIDTH_OES, &backingWidth);
+	glGetRenderbufferParameterivOES(GL_RENDERBUFFER_OES,
+			GL_RENDERBUFFER_HEIGHT_OES, &backingHeight);
+
+	OS::VideoMode vm;
+	vm.fullscreen = true;
+	vm.width = backingWidth;
+	vm.height = backingHeight;
+	vm.resizable = false;
+	return vm;
+}
 
-	int remaining = 0;
-	for (int i = 0; i < max_touches; i++) {
+@interface GLView ()
 
-		if (touches[i] == NULL)
-			continue;
-		if (touches[i] == p_touch)
-			touches[i] = NULL;
-		else
-			++remaining;
-	};
-	return remaining;
-};
+@property(nonatomic, strong) CADisplayLink *displayLink;
+@property(nonatomic, assign) BOOL useCADisplayLink;
 
-static void clear_touches() {
+@end
 
-	for (int i = 0; i < max_touches; i++) {
+@implementation GLView
 
-		touches[i] = NULL;
-	};
-};
+@synthesize animationInterval = _animationInterval;
```

Comment:
Since we are over-riding *both* the setter and the getter for `animationInterval` we must synthesize it to another name to use internally in both of those methods.